### PR TITLE
Frontend enhancement

### PR DESCRIPTION
**Pull request by chudy1997**

### CODE REVIEW

File: src/components/Steps/Criteria/custom.js [hunk 1]
```diff
@@ -50,46 +59,74 @@ class CustomCriteria extends React.Component {
...
-                                    <Checkbox className="input" field="ascending" defaultValue={ascending} validate={() => ({})} />
-                                </div>
-                            </div>
-                        </form>
-                    )}
-                </Form>
+                <div className="line">
+                    <label>Url</label>
+                    <input
+                        className={classnames('input', { error: this.state.urlError })}
```
Comment (franczak) [inline src/components/Steps/Criteria/custom.js]: const

File: src/components/Steps/Criteria/custom.js [hunk 2]
```diff
@@ -50,46 +59,74 @@ class CustomCriteria extends React.Component {
...
+                        onChange={({ target: { value } }) => {
+                            this.setState({ urlError: this.validateUrl(value) });
+                            this.props.updateCriteriaData({ url: value });
+                        }}
+                    />
+                </div>
+                <div className="line">
+                    <label>Property access</label>
+                    <input
+                        className={classnames('input', { error: this.state.propertyAccessError })}
```
Comment (franczak) [inline src/components/Steps/Criteria/custom.js]: const

File: src/components/Steps/Criteria/custom.js [hunk 3]
```diff
@@ -50,46 +59,74 @@ class CustomCriteria extends React.Component {
...
+                </div>
+                <div className="line">
+                    <label>Importance</label>
+                    <InputRange
+                        className="input"
+                        minValue={1}
+                        maxValue={100}
+                        step={1}
+                        value={importance}
+                        onChange={imp => this.props.updateCriteriaData({ importance: imp })}
```
Comment (franczak) [inline src/components/Steps/Criteria/custom.js]: importance instead of imp wouldnt be better?

Comment (chudy1997) [reply src/components/Steps/Criteria/custom.js]: I had to use imp since importance is already defined in upper scope (ESLint).

**Pull request merged**